opacclient/libopac/src/main/java/de/geeksfactory/opacclient/apis/OpacApi.java

```diff
@@ -86,6 +86,11 @@
     public static final int SUPPORT_FLAG_WARN_RESERVATION_FEES = 0x0000020;
 
     /**
+     * Asks the user responsibly about prolong fees
+     */
+    public static final int SUPPORT_FLAG_WARN_PROLONG_FEES = 0x0000030;
```

Comment:
This is used by a bitmask and therefore needs to be a value that only has one bit set, just choose ``0x0000040``.